gqcpy/src/ONVBasis/SpinUnresolvedONVBasis_bindings.cpp

```diff
@@ -52,35 +52,33 @@ void bindSpinUnresolvedONVBasis(py::module& module) {
 
         .def(
             "arcWeight",
-            [](const SpinUnresolvedONVBasis& onv_basis, size_t p, const size_t n) {
-                return onv_basis.arcWeight(p, n);
-            },
+            &SpinUnresolvedONVBasis::arcWeight,
             py::arg("p"),
             py::arg("n"),
             "Return the arc weight of the arc starting at a vertex (p, n), with p the orbital index and n the electron index.")
 
-        .def(
+        .def_static(
             "calculateDimension",
-            [](const SpinUnresolvedONVBasis& onv_basis, size_t M, const size_t N) {
-                return onv_basis.calculateDimension(M, N);
-            },
+            &SpinUnresolvedONVBasis::calculateDimension,
             py::arg("M"),
             py::arg("N"),
             "Calculate the dimension of a spin-unresolved ONV with M spinors and N electrons.")
```

Comment:
Why should this become a `.def_static`?

Reply:
Since its declaration has the `static` keyword `static size_t calculateDimension(const size_t M, const size_t N);`